languages/cpp/generic/cpp_to_generic.mli

```diff
@@ -1,3 +1,7 @@
 (* may raise AST_generic.Error *)
 val program : Ast_cpp.program -> AST_generic.program
-val any : Ast_cpp.any -> AST_generic.any
+
+type cpp_parsing_option = [ `AsFunDef | `AsVarDefWithCtor ]
```

Comment:
maybe add a comment there or at least a reference to Rule_options.atd so people can understand what are those parameters for.

Reply:
Done